### ISSUE TITLE
When selecting Host through UI, unable to Add Data for Meta-Data section when there are no previous key-value pairs

### ISSUE THREAD

**Issue by jasonklotzer** [realvnc-labs/rport#1188]
**Describe the bug**
Through the UI, when a new host is added and I go to the "Meta-Data" section, there is no longer a "+ Add Data" button on the top right that shows up. This only happens when there is no metadata for the given host. 

**Environment**
rport-frontend-0.9.12-16-build-1144

**Log file**
N/A - Applies to UI

**Screenshots**
As you can see the "+ Add Data" button does not  show up when there's no existing metadata:
![image](https://github.com/realvnc-labs/rport/assets/2832232/a468002e-a40f-412b-a10c-bdcbfd63f065)

But when there is existing metadata, then the button shows up:
![image](https://github.com/realvnc-labs/rport/assets/2832232/90c32822-7530-4f2d-8fa1-e866cca5b4bd)

**Comment by swilson563** [comment]
Same issue here.

**Comment by thorstenkramm** [comment]
This issue has been solved.

Try updating the frontend to the latest version [frontend 0.9.12-17](https://download.rport.io/frontend/stable/) or newer, which contains a fix for this issue.


```
curl -L https://download.rport.io/frontend/stable/latest.php -o /tmp/rport-frontend-latest.zip
sudo -u rport sh -c "cd /var/lib/rport/docroot; unzip -o /tmp/rport-frontend-latest.zip"
rm /tmp/rport-frontend-latest.zip
cat /var/lib/rport/docroot/version.txt
```
 
Hard reload (Shift-Reload) your browser. _No need to restart the rport server._

🙏 Much appreciated if you confirm your issue has been solved by the update.

**Comment by jasonklotzer** [comment]
Can confirm this is fixed in `rport-frontend-0.9.12-17-build-1145`. Thank you!
